BUILD.md

```diff
@@ -0,0 +1,20 @@
+# Build
+
+## Building subpackages and uploading to pypi
+1. Within a subpackage, make sure the version is updated in `pyproject.toml`.
+2. Install prerequisites:
+```bash
+python -m pip install --upgrade pip
+python -m pip install --upgrade build twine
+```
+3. Build the distribution within the subpackage. For example, to build the `fixedpointmath` package,
+```bash
+cd lib/fixedpointmath
+python -m build
+```
+4. Upload to pypi. The below command will ask for a username and password.
+```bash
+python -m twine upload --repository pypi dist/*
+``` 
+
+TODO figure out how to expand access across one current account (currently `fixedpointmath` is published under `slundquist`)
```

Comment:
uh oh, I hope we can transfer that.

Reply:
Does look like there's ways to add/remove collaborators, as well as transferring ownership. Just need pypi accounts.

![image](https://github.com/delvtech/agent_0/assets/1431723/9d1544de-673f-4005-930a-f0e1f7973d43)